### DISCUSSION TITLE
Can't connect Crazyflie to cfclient with crazyradio (but can to bootload?)

### DISCUSSION THREAD

**Discussion by Simply-Spinach** [bitcraze/discussions#1712] [Q&A] [answered]
I'm struggling right now because I can't get the crazyflie connected to the cfclient.  I've tried different addresses and trying to connect the radio before the drone and after the drone and all that and nothing seems to work.  I am developing on it and am able to have it enter bootloader mode and 'make cload' it no problem.

Some more things you may want to know:
     - I'm using Windows to run the cfclient and WSL to flash/develop it.
     - I do notice when flashing the crazyflie that I need to disconnect and reconnect the radio between attempts or otherwise the radio doesn't connect to the drone.  That trick works to detect the Crazyflie in the cfclient, but not to connect afterward.

I've tried before:
    - Updating the driver on the radio.  It's currently flashed with libusb-win32 (v1.4.0.0)
    - The unplugging and plugging back in trick that works with cload (only allows the radio to see the crazyflie and when I try to connect, it doesn't connect )
    - Changing the drone's address

If anyone knows how to fix this, that would be great.  I'm really stuck at this point and I would appreciate any help at this point.

**Comment by ToveRumar** [comment]
Hi @PapercutSmothie!

This sounds strange. Are you having any issues connecting over Usb or is it only via radio?

Can you see any clues from the cfclient logs?

**Comment by Simply-Spinach** [comment, marked as answer]
Sorry for the late response.  I found out that the problem was that when I put the .uf2 onto the radio, something failed that made it unstable.  Reuploading the .uf2 fixed it.
